addons/resource.language.en_gb/resources/strings.po

```diff
@@ -14143,7 +14143,7 @@ msgstr ""
 #. Description of setting "Appearance -> File lists -> Show file extensions" with label #497
 #: system/settings/settings.xml
 msgctxt "#36123"
-msgid "Show file extensions on media files. For example, 'You Enjoy Myself.mp3' would be simply be shown as 'You Enjoy Myself'."
+msgid "Show file extensions on media files. For example, 'You enjoy myself.mp3' would be simply be shown as 'You enjoy myself'."
```

Comment:
It's a common naming convention for songs to use camel case, so I'd drop this change.